### ISSUE TITLE
Identify Breadcrumbs - Front end (Web)

### ISSUE THREAD

**Issue by ZoeSimon** [bcgov/lucy-web#505]
**Describe the task**
Identify Breadcrumbs for front end (web)

Breadcrumbs in this context include: "This exists, it is blocked, here is why, and this is what to do when it is unblocked"

**Comment by ghost** [comment]
https://docs.google.com/document/d/1rF9MTJzGIFVv1Ds0yX4TiwAvE9PTJvPscbyxhCWW7-s/edit?usp=sharing

**Comment by ZoeSimon** [comment]
Mike Wells' comment from ticket #495 

- what things are partly done
- what low priority stuff has no tests, but needs doc
- things that were not done, but anticipated to be needed later, and advice on changes that would needed to be made, such as:
- what things would need to change if we moved to high availability i.e. more than one pod running your api.

**Comment by andrea-williams** [comment]
I've taken a look at this doc and added a few extra things I thought of. I'm happy with the current state of it - it probably hasn't captured everything, but it's difficult to remember everything when doing a brain dump.

I strongly recommend that the Google Doc for this gets converted to a markdown file so that it can be added to our github repo directly before @rajfreshworks leaves.

**Comment by ghost** [comment]
Thanks for doing that @andrea-williams Right now, the docs that are reviewed and finalized are stored in our repo in .doc format. I'll store it as a markdown instead since that makes it much easier to maintain
